packages/xo-web/src/xo-app/about/index.js

```diff
@@ -34,42 +31,87 @@ export default class About extends Component {
     serverVersion.then(serverVersion => {
       this.setState({ serverVersion })
     })
+
+    if (process.env.XOA_PLAN > 4 && COMMIT_ID !== '') {
+      getMasterCommit()
+        .then(async commit => {
```

Comment:
You can use `async/await` here as well.

Reply:
`componentWillMount` is called before the first render. Using `async/await` here will not delay the first render?

Reply:
`.then()` and `async/await` are equivalent.

But yes, indeed, the first render won't have the GitHub data, and it needs to be properly handled (I thought that's what you were doing with `state.fetched` but actually, you're showing an error message when it's `false`, so maybe change that too). And then it will render again with the data when you call `setState`. But switching from `.then()` to `async/await` won't affect that workflow, in both cases there will be a small delay before the status is shown.